ext/DynaLoader/DynaLoader_pm.PL

```diff
@@ -484,6 +484,9 @@ sub dl_findfile {
 	    <<$^O-eq-cygwin>>
             push(@names,"cyg$_.$dl_so")  unless m:/:;
 	    <</$^O-eq-cygwin>>
+	    <<$^O-eq-msys>>
```

Comment:
This commit should also update the version to `'1.55'`

Reply:
Done